src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const gallery = document.getElementsByClassName('gallery')[0];
+
+gallery.addEventListener('click', (galleryEvent) => {
+  galleryEvent.preventDefault();
+
+  if (galleryEvent.target.classList.contains('gallery__large-img')
+    || galleryEvent.target.classList.contains('gallery')
+    || galleryEvent.target.classList.contains('gallery__list')
+    || galleryEvent.target.classList.contains('list-item__link')
+    || galleryEvent.target.tagName === 'DIV') {
+    return;
+  }
```

Comment:
there should be no this if checking, if there would be additional elements, you will add each of them to this `if` also?
use `target.closest` to find closest `a` of the element you've clicked